docs/source/admin/capacity-planning.md

```diff
@@ -0,0 +1,246 @@
+
+In the strictest, safest case, you can have these two numbers be the same.
+That means that each user is _limited_ to fit within the resources allocated to it.
+This avoids **[oversubscription](oversubscription)** of resources (allowing use of more than you have available),
+at the expense (in a literal, this-costs-money sense) of reserving lots of usually-idle capacity.
+
+But when deploying JupyterHub,
+you will likely find that a relatively small fraction of users use lots more resources than others,
+making oversubscription attractive (to a point).
+
+Having a gap between the request and the limit means you can fit a number of _typical_ users on a node (based on the request),
+but still limit how much a runaway user can gobble up for themselves.
+
+(oversubscription)=
+
+### Oversubscribed CPU is okay, running out of memory is bad
+
+An important consideration when assigning resources to users is:
+
+> What happens when users need more than I've given them?
+
+A good summary to keep in mind: **when tasks don't get enough CPU, things are slow.
+When they don't get enough memory, things are broken.**
+Which means it's very important that users have enough memory,
+but much less important that they always have exclusive access to all the CPU they can use.
+
+This relates to [Limits and Requests](limits-requests),
+because these are the consequences of your limits and/or requests not matching what users actually try to use.
+
+A table of mismatched resource allocation situations and their consequences:
+
+| issue                                                    | consequence                                                                           |
+| -------------------------------------------------------- | ------------------------------------------------------------------------------------- |
+| Requests too high                                        | Unnecessarily high cost and/or low capacity.                                          |
+| CPU limit too low                                        | Poor performance experienced by users                                                 |
+| CPU oversubscribed (too-low request + too-high limit)    | Poor performance across the system; may crash, if severe                              |
+| Memory limit too low                                     | Servers killed by Out-of-Memory Killer (OOM); lost work for users                     |
+| Memory oversubscribed (too-low request + too-high limit) | System memory exhaustion - all kinds of hangs and crashes and weird errors. Very bad. |
+
+Note that the 'oversubscribed' problem case is where the request is lower than _typical_ usage,
+meaning that the total reserved resources isn't enough for the total _actual_ consumption.
+This doesn't mean that _all_ your users exceed the request,
+just that the _limit_ gives enough room for the _average_ user to exceed the request.
+
+### Example case for oversubscribe memory
```

Comment:
Maybe we should point out somewhere really clearly that the most important number is the **Average CPU / Memory used per user on a node**. That's ultimately the number that decides whether you hit your node resource limits

Reply:
```suggestion
### Example case for oversubscribing memory
```